.travis.yml

```diff
@@ -17,13 +24,17 @@ matrix:
     - php: 7.1
     - php: 7.2
     - php: hhvm
+    - stage: release
+      if: branch = master
+      php: 5.3
+      dist: precise
```

Comment:
this one should use `composer install --no-dev -o` as install script

Reply:
indeed thanks